### PR TITLE
Added new programs to learning and exercises packages

### PR DESCRIPTION
**Pull request by marekk1992**

### CODE REVIEW

File: src/exercises/equalityPrinter/EqualityPrinter.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package exercises.equalityPrinter;
+
+public class EqualityPrinter {
+
+    public static void printEqual(int number1, int number2, int number3) {
+        if ((number1 < 0) || (number2 < 0) || (number3 < 0)) {
```
Comment (RobertasVainickas) [inline src/exercises/equalityPrinter/EqualityPrinter.java:6]: Unnecessary inner braces. 
Please fix all the other places in this PR too as there are quite a lot of them

File: src/exercises/teenNumberChecker/TeenNumberChecker.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package exercises.teenNumberChecker;
+
+public class TeenNumberChecker {
+
+    public static boolean hasTeen(int value1, int value2, int value3) {
+        return (value1 <= 19 && value1 >= 13) || (value2 <= 19 && value2 >= 13)
```
Comment (RobertasVainickas) [inline src/exercises/teenNumberChecker/TeenNumberChecker.java:6]: You introduced isTeen method on the line 10, but don't use it here

**Review by RobertasVainickas — CHANGES_REQUESTED** [review]

File: src/exercises/equalityPrinter/EqualityPrinter.java (hunk 1 above)
Comment (marekk1992) [reply src/exercises/equalityPrinter/EqualityPrinter.java]: Removed inner braces from all required places.

File: src/exercises/teenNumberChecker/TeenNumberChecker.java (hunk 2 above)
Comment (marekk1992) [reply src/exercises/teenNumberChecker/TeenNumberChecker.java]: Added isTeen method to hasTeen method.

File: src/learning/secondsAndMinutesChallenge/SecondsAndMinutesChallenge.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    public static void main(String[] args) {
+
+        System.out.println(getDurationString(65, 45));
+        System.out.println(getDurationString(3945L));
+        System.out.println(getDurationString(-41));
+        System.out.println(getDurationString(65, 9));
+    }
+
+    private static String getDurationString(long minutes, long seconds) {
+        if ((minutes < 0) || (seconds < 0) || (seconds > 59)) {
```
Comment (RobertasVainickas) [inline src/learning/secondsAndMinutesChallenge/SecondsAndMinutesChallenge.java:16]: You left braces here

File: src/learning/methodOverloading/MethodOverloading.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        return score * 1000;
+    }
+
+    public static int calculateScore() {
+        System.out.println("No player name, no player score");
+        return 0;
+    }
+
+    public static double calcFeetAndInchesToCentimeters(double feet, double inches) {
+        if ((feet < 0) || (inches < 0) || (inches > 12)) {
```
Comment (RobertasVainickas) [inline src/learning/methodOverloading/MethodOverloading.java:34]: And here

**Review by RobertasVainickas — CHANGES_REQUESTED** [review]

File: src/learning/secondsAndMinutesChallenge/SecondsAndMinutesChallenge.java (hunk 3 above)
Comment (marekk1992) [reply src/learning/secondsAndMinutesChallenge/SecondsAndMinutesChallenge.java]: Removed

File: src/learning/methodOverloading/MethodOverloading.java (hunk 4 above)
Comment (marekk1992) [reply src/learning/methodOverloading/MethodOverloading.java]: Removed

**Review by RobertasVainickas — APPROVED** [review]

**Pull request merged**